src/x/__tests__/xMaxIndex.test.ts

```diff
@@ -6,12 +6,21 @@ describe('array-xMaxIndex', () => {
   typedArray[1] = 2;
   typedArray[2] = 3;
 
+  let anotherArray = new Uint16Array(4);
```

Comment:
You can directly create the array using

`let anotherArray = Uint16Array.from([3,5,4,3]);`